roles/matrix-jitsi/defaults/main.yml

```diff
@@ -266,4 +270,4 @@ matrix_jitsi_jvb_container_rtp_tcp_host_bind_port: "{{ matrix_jitsi_jvb_rtp_tcp_
 # Controls whether the matrix-jitsi-jvb container exposes its Colibri WebSocket port (tcp/9090 in the container).
 #
 # Takes an "<ip>:<port>" or "<port>" value (e.g. "127.0.0.1:12090"), or empty string to not expose.
-matrix_jitsi_jvb_container_colibri_ws_host_bind_port: ''
+matrix_jitsi_jvb_container_colibri_ws_host_bind_port: 9090
```

Comment:
Oh.. why are we exposing these by default now?

Reply:
Ah it's needed on the additional jvb as it's outside the docker network but you're right it shouldn't be exposed on the matrix host and hence shouldn't be a default. Good catch, i'll change.